common/network-common/src/main/java/org/apache/spark/network/server/ChunkFetchRequestHandler.java

```diff
@@ -65,17 +52,8 @@ public ChunkFetchRequestHandler(
     this.maxChunksBeingTransferred = maxChunksBeingTransferred;
   }
 
-  @Override
-  public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws Exception {
-    logger.warn("Exception in connection from " + getRemoteAddress(ctx.channel()), cause);
-    ctx.close();
-  }
-
-  @Override
-  protected void channelRead0(
-      ChannelHandlerContext ctx,
-      final ChunkFetchRequest msg) throws Exception {
-    Channel channel = ctx.channel();
+  public void processFetchRequest(
+      final Channel channel, final ChunkFetchRequest msg) throws Exception {
```

Comment:
shall we define channel as a protected member variable?

Reply:
It's OK for `TransportRequestHandler`, but for `ChunkFetchChannelHandler`, maybe get the current channel from `ChannelHandlerContext` is a safer way.